package.json

```diff
@@ -88,9 +88,9 @@
     "@metamask/eth-sig-util": "^5.0.2",
     "@nomicfoundation/hardhat-toolbox": "^2.0.0",
     "@nomiclabs/hardhat-ethers": "^2.2.1",
-    "@rsksmart/rif-relay-client": "github:rsksmart/rif-relay-client",
+    "@rsksmart/rif-relay-client": "github:rsksmart/rif-relay-client#fix/verify_with_relay_hub",
     "@rsksmart/rif-relay-contracts": "github:rsksmart/rif-relay-contracts",
-    "@rsksmart/rif-relay-server": "github:rsksmart/rif-relay-server",
+    "@rsksmart/rif-relay-server": "github:rsksmart/rif-relay-server#fix/verify_with_relay_hub",
```

Comment:
We need to update it after merging the PRs

Reply:
updated